src/verify/VerificationCodeInputScreen.tsx

```diff
@@ -92,7 +107,9 @@ function VerificationCodeInputScreen({
           inputPlaceholder={t('phoneVerificationInput.codeInputPlaceholder')}
           onInputChange={setCode}
           shouldShowClipboard={(content) =>
-            !!content && content.length === VERIFICATION_CODE_LENGTH
+            Platform.OS === 'android' &&
```

Comment:
on second thought we don't really need to restrict it on android - i thought that we don't need to show the paste button on ios because we do the auto populate thing but it doesn't really hurt to have it. i removed the check